release_manifests/24.10-0.7.0.0-0.yaml

```diff
@@ -0,0 +1,45 @@
+precompiled:
+- 24.10-0.7.0.0-0-5.15.0-25-generic-ubuntu22.04-amd64
+- 24.10-0.7.0.0-0-5.15.0-25-generic-ubuntu22.04-arm64
+- 24.10-0.7.0.0-0-6.8.0-31-generic-ubuntu24.04-amd64
+- 24.10-0.7.0.0-0-6.8.0-31-generic-ubuntu24.04-arm64
+- 24.10-0.7.0.0-0-6.8.0-1016-oracle-ubuntu22.04-amd64
+- 24.10-0.7.0.0-0-6.8.0-1016-oracle-ubuntu22.04-arm64
+- 24.10-0.7.0.0-0-6.8.0-1016-oracle-ubuntu24.04-amd64
+- 24.10-0.7.0.0-0-6.8.0-1016-oracle-ubuntu24.04-arm64
+- 24.10-0.7.0.0-0-6.8.0-1018-azure-ubuntu22.04-amd64
+- 24.10-0.7.0.0-0-6.8.0-1018-azure-ubuntu22.04-arm64
+- 24.10-0.7.0.0-0-6.8.0-1018-azure-ubuntu24.04-amd64
+- 24.10-0.7.0.0-0-6.8.0-1018-azure-ubuntu24.04-arm6
+- 24.10-0.7.0.0-0-6.8.0-1019-aws-ubuntu22.04-amd64
+- 24.10-0.7.0.0-0-6.8.0-1019-aws-ubuntu22.04-arm64
+- 24.10-0.7.0.0-0-6.8.0-1019-aws-ubuntu24.04-amd64
+- 24.10-0.7.0.0-0-6.8.0-1019-aws-ubuntu24.04-arm64
+dynamically_compiled:
+- 24.10-0.7.0.0-0-rhcos4.11-amd64
+- 24.10-0.7.0.0-0-rhcos4.12-amd64
+- 24.10-0.7.0.0-0-rhcos4.13-amd64
+- 24.10-0.7.0.0-0-rhcos4.13-arm64
+- 24.10-0.7.0.0-0-rhcos4.14-amd64
+- 24.10-0.7.0.0-0-rhcos4.14-arm64
+- 24.10-0.7.0.0-0-rhcos4.15-amd64
+- 24.10-0.7.0.0-0-rhcos4.15-arm64
+- 24.10-0.7.0.0-0-rhcos4.16-amd64
+- 24.10-0.7.0.0-0-rhcos4.16-arm64
+- 24.10-0.7.0.0-0-rhel8.6-amd64
+- 24.10-0.7.0.0-0-rhel8.6-arm64
+- 24.10-0.7.0.0-0-rhel8.8-amd64
+- 24.10-0.7.0.0-0-rhel8.8-arm64
+- 24.10-0.7.0.0-0-rhel8.9-amd64
+- 24.10-0.7.0.0-0-rhel8.9-arm64
+- 24.10-0.7.0.0-0-rhel8.10-amd64
+- 24.10-0.7.0.0-0-rhel8.10-arm64
+- 24.10-0.7.0.0-0-rhel9.0-amd64
```

Comment:
rhel9.0 isn't supported

Reply:
tags acquired with `ngc registry image list nvcr.io/nvstaging/mellanox/doca-driver:24.10-0.7.0.0-0* | jq -r '.[].tag' | sort -V | grep '64$'`... i'll remove it

Reply:
/nvstaging/mellanox has a lot of images which should not be published to a public repo. Please, align it with supported matrix

Reply:
Ok.
But the [support matrix](https://confluence.nvidia.com/display/NetworkingBU/2024-rel2#tab-DOCA%2FOFED+Container+support+Matrix) doesn't state architectures for RHEL... so does that imply _both_? or only AMD64?

Reply:
RHEL only AMD64

Reply:
ok. should be good now, please verify me!